modules/Bio/EnsEMBL/Compara/PipeConfig/EBI/Vertebrates/StrainsReindexMembers_conf.pm

```diff
@@ -1,112 +0,0 @@
-    -prev_tree_db murinae_nctrees_prev -collection murinae -member_type ncrna
-    -prev_tree_db sus_ptrees_prev      -collection sus     -member_type protein
-    -prev_tree_db sus_nctrees_prev     -collection sus     -member_type ncrna
-
-e98 protein-trees
-    -mlss_id 40128 -member_type ncrna   -prev_rel_db murinae_nctrees_prev $(mysql-ens-compara-prod-7-ensadmin details hive)
-e98 ncRNA-trees
-    -mlss_id 40126 -member_type protein -prev_rel_db murinae_ptrees_prev  $(mysql-ens-compara-prod-7-ensadmin details hive)
-
-e94 protein-trees
-    -mlss_id 40111 -member_type protein -member_db $(mysql-ens-compara-prod-2 details url waakanni_load_members_94) -prev_rel_db $(mysql-ens-compara-prod-1 details url mateus_murinae_protein_trees_93) $(mysql-ens-compara-prod-1-ensadmin details hive)
-e94 ncRNA-trees
-    -mlss_id 40112 -member_type ncrna -member_db $(mysql-ens-compara-prod-2 details url waakanni_load_members_94) -prev_rel_db $(mysql-ens-compara-prod-1 details url mateus_murinae_ncrna_trees_93) $(mysql-ens-compara-prod-1-ensadmin details hive)
-
-e93 protein-trees
-    -mlss_id 40111 -member_type protein -member_db $(mysql-ens-compara-prod-2 details url mateus_load_members_93) -prev_rel_db $(mysql-ens-compara-prod-3 details url carlac_murinae_reindex_protein_92)
-e93 ncRNA-trees
-    -mlss_id 40112 -member_type ncrna -member_db $(mysql-ens-compara-prod-2 details url mateus_load_members_93) -prev_rel_db $(mysql-ens-compara-prod-2 details url muffato_murinae_ncrna_trees_92)
-
-e91 protein-trees
-    -mlss_id 40111 -member_type protein -member_db $(mysql-ens-compara-prod-2 details url mateus_load_members_91) -prev_rel_db $(mysql-ens-compara-prod-3 details url carlac_murinae_protein_trees_90)
-e91 ncRNA-trees
-    -mlss_id 40112 -member_type ncrna -member_db $(mysql-ens-compara-prod-2 details url mateus_load_members_91) -prev_rel_db $(mysql-ens-compara-prod-4 details url mateus_murinae_nctrees_90)
-
-=head1 DESCRIPTION
-
-A specialized version of ReindexMembers_conf to use in Vertebrates for
-the mouse-strains, although "murinae" is only used to set up the
-pipeline name.
-
-=head1 AUTHORSHIP
-
-Ensembl Team. Individual contributions can be found in the GIT log.
-
-=head1 APPENDIX
-
-The rest of the documentation details each of the object methods.
-Internal methods are usually preceded with an underscore (_)
-
-=cut
-
-package Bio::EnsEMBL::Compara::PipeConfig::EBI::Vertebrates::StrainsReindexMembers_conf;
-
-use strict;
-use warnings;
-
-use base ('Bio::EnsEMBL::Compara::PipeConfig::ReindexMembers_conf');
-
-sub default_options {
-    my ($self) = @_;
-    return {
-        %{$self->SUPER::default_options},
-
-        'division'      => 'vertebrates',
-
-        # Main capacity for the pipeline
-        'copy_capacity'                 => 4,
-
-        # Params for healthchecks;
-        'hc_capacity'                     => 40,
-        'hc_batch_size'                   => 10,
-
```

Comment:
Where are these capacities being set now?

Reply:
They are in `modules/Bio/EnsEMBL/Compara/PipeConfig/ReindexMembers_conf.pm` lines 111, 114 and 115, respectively.